drake/solvers/Optimization.h

```diff
@@ -469,17 +469,17 @@ class DRAKEOPTIMIZATION_EXPORT OptimizationProblem {
     }
     return p;
   }
-  
-  size_t getNumVars() const { return num_vars; }
+
+  size_t get_num_vars() const { return num_vars; }
   const Eigen::VectorXd& getInitialGuess() const { return x_initial_guess; }
```

Comment:
Not sure what you mean.  Does Google Style disfavor get_foo as the foo accessor?